apps/transport/test/db/db_dataset_test.exs

```diff
@@ -210,7 +210,7 @@ defmodule DB.DatasetDBTest do
           })
         end)
 
-      assert logs =~ "error while importing dataset: %{legal_owner_company_siren:"
+      assert logs =~ ~r/error while importing dataset(.*)legal_owner_company_siren/
```

Comment:
Le logging "dumpe" la map directement, avant l'ordre semblait faussement garanti (assez pour que ça sorte tout le temps dans le même ordre dans les tests), et now it's gone, il faut introduire de la souplesse, ce que je fais ici.